indico/modules/designer/models/templates.py

```diff
@@ -39,6 +41,12 @@ class DesignerTemplate(db.Model):
         db.String,
         nullable=False
     )
+    registration_form_id = db.Column(
+        db.Integer,
```

Comment:
not needed, `db.ForeignKey` looks up the referenced type automatically (which nobody knew early on so in most places we explicitly repeat it)